hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusUpdateTopicHandler.java

```diff
@@ -188,8 +223,6 @@ private void resolveMutableBuilderAttributes(
             } else {
                 builder.adminKey(key);
             }
-        } else {
-            builder.adminKey(topic.adminKey());
```

Comment:
Since the change [here](https://github.com/hashgraph/hedera-services/pull/15350/files#diff-7b1638b73c6ed756b751fd44e9f1dea1d91becd47cf0854c3b55181b8d50c859R158) the builder is one to one to the topic. There is no need to copy over the values again. Also, I changed the `resolvedUpdateMetaFrom` method(called below)  to expect `ExpiryMeta` instead of `Topic` and build the `ExpiryMeta` before the call. That way we don't need to pass `Topic`